Neos.ContentRepositoryRegistry/Classes/Command/ContentStreamCommandController.php

```diff
@@ -33,6 +33,11 @@ class ContentStreamCommandController extends CommandController
      */
     public function pruneCommand(string $contentRepository = 'default', bool $removeTemporary = false): void
     {
+        if (!$this->output->askConfirmation(sprintf('> This operation in "%s" cannot be reverted. Are you sure to proceed? (y/n) ', $contentRepository), false)) {
```

Comment:
@mhsdesign What was the intention on adding this unskippable confirmation? I remember using the `pruneCommand` and `pruneRemovedFromEventStreamCommand` regularly to cleanup the database.

Reply:
Hm. But do we really assume that this commands will only ever have the need to be triggered manually? Cleaning up the contentstream tables of junk data is a bit of a different use case then `cr:prune` and `cr:projectionReplay`. 

And I'm worried about adding this change now in a feature PR without addressing. This might be a change someone running an active Neos 9 project would like to see in the Logs.

Reply:
Imho this must be able to be part of a CI process, so at least a confirmation parameter must be added to the CLI call.